src/js/components/spinner.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import classnames from "classnames";
+
+const BASE_CLASS = "mw-spinner";
+
+export default class Spinner extends React.Component {
+
+	render() {
+		const classes = [
+			BASE_CLASS,
+			this.props.className
+		];
+
+		const style = { backgroundColor: this.props.color };
```

Comment:
This is only for the background. the default color is #fff set as a prop. Did that in order to make it possible to use it in various places with different colors without having to have insight on the insights of the spinner HTML/CSS. reasonable?